.github/workflows/node.js.yml

```diff
@@ -23,6 +23,6 @@ jobs:
       - name: Deploy 🚀
         uses: JamesIves/github-pages-deploy-action@releases/v3
         with:
-          ACCESS_TOKEN: ${{ secrets.ACCESS_TOKEN }}
+          ACCESS_TOKEN: ${{ secrets.ACCESS_TOKEN }} # Use a different Personal Access token personal access token you generated with at least public_repo, read:org, workflow access by adding it to the Settings/Secrets part of this repository with anme ACCESS_TOKEN
```

Comment:
Change this to
Use the generated personal access token with public_repo, read:org, workflow access and add it to repository secrets with the name OPEN_SOURCE_TOKEN

Reply:
Done